src/main/java/net/neoforged/neoforge/event/EventHooks.java

```diff
@@ -1120,4 +1123,70 @@ public static MobSplitEvent onMobSplit(Mob parent, List<Mob> children) {
         NeoForge.EVENT_BUS.post(event);
         return event;
     }
+
+    /**
+     * Fires the {@link ProjectileWeaponLoadEvent.Pre}. Returns if the event is cancelled.
+     * 
+     * @param entity         The living entity using the projectile weapon.
+     * @param weaponItem     The projectile weapon item stack.
+     * @param projectileItem The projectile weapon item stack.
+     * @param hand           The hand helding the projectile weapon.
+     * @param canLoad        If the projectile weapon can start loading.
+     * @return If the event is cancelled.
```

Comment:
Wait I thought this event was not cancellable...

Reply:
Welp, it sure is not cancellable, should be the same as `@param canLoad`'s description :P